apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/step-based-flow.tsx

```diff
@@ -342,17 +342,29 @@ export const StepBasedFlow: FunctionComponent<AuthenticationFlowPropsInterface>
         }
 
         const isValid: boolean = validateStepAddition(authenticator, steps[ stepIndex ].options);
+        const isFirstFactorAuth: boolean =
+            ApplicationManagementConstants.FIRST_FACTOR_AUTHENTICATORS.includes(authenticatorId);
+        const isSecondFactorAuth: boolean =
+            ApplicationManagementConstants.SECOND_FACTOR_AUTHENTICATORS.includes(authenticatorId);
+        const isValidSecondFactorAddition: boolean = SignInMethodUtils.isSecondFactorAdditionValid(
+            authenticator.defaultAuthenticator.authenticatorId,
```

Comment:
```suggestion
            authenticator?.defaultAuthenticator?.authenticatorId,
```

Reply:
Resolved in https://github.com/wso2/identity-apps/pull/4811/commits/1d8bf5a413906bf2f70fc933ec9bdc8519367a2e